AdyenCard/Utilities/BinLookup/BinLookupRequest.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright (c) 2020 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal struct BinLookupRequest: Request {
+    
+    internal typealias ResponseType = BinLookupResponse
+    
+    internal var path: String { "checkoutshopper/v1/bin/binLookup?token=\(clientKey)" }
```

Comment:
the path doesn't have to have `token=\(clientKey)` its injected by the environment

Reply:
👍